src/arrayMethodSort.js

```diff
@@ -15,23 +15,23 @@ const compareAsString = (a, b) => {
 
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction = compareAsString) {
-    let count = 0;
+    let hasChanged = 0;
```

Comment:
Булевого типу змінна має бути

Reply:
Ось тут можеш не задавати значення
```suggestion
    let hasChanged;
```